### PR TITLE
Vtk and visit tweaks

### PR DESCRIPTION
**Pull request by opadron**
Fixes a few issues in the VTK and VISIT packages.

### CODE REVIEW

File: var/spack/repos/builtin/packages/vtk/package.py [hunk 1]
```diff
@@ -136,9 +134,17 @@ def cmake_args(self):
 
         if '+mpi' in spec:
             cmake_args.extend([
+                ''.join((
```
Comment (chuckatkins) [inline var/spack/repos/builtin/packages/vtk/package.py]: Should these be added here or in `setup_environment(...)`?

Comment (chuckatkins) [reply var/spack/repos/builtin/packages/vtk/package.py]: ping?

**Comment by opadron** [comment]
@chuckatkins ping

**Comment by adamjstewart** [comment]
Ping @chuckatkins @cyrush @danlipsa 

@opadron can you rebase if this is still something you want?

**Comment by alalazo** [comment]
@opadron @chuckatkins I'm closing this but feel free to reopen if you think it is still needed.

**Pull request closed without merging**